src/utils/utils.ts

```diff
@@ -0,0 +1,60 @@
+import { Request } from 'express-serve-static-core';
+import * as fs from 'fs';
+import {
+  errorFile500,
+  errorNotFound404,
+  successFile200OK,
+  successFileChange205,
+} from '../constants.js';
```

Comment:
js?

Reply:
I know, it looks weird, but it is the best solution I managed to find. TS ignores these .js extensions but compiled js files use it.
(I found this here: https://stackoverflow.com/questions/56723678/node-experimental-modules-error-cannot-find-module )